tests/zkevm/test_worst_opcode.py

```diff
@@ -39,6 +40,7 @@
 )
 @pytest.mark.parametrize("empty_topic", [True, False])
 @pytest.mark.parametrize("fixed_offset", [True, False])
+@pytest.mark.parametrize("non_zero_data", [True, False])
```

Comment:
I thin we should probably remove this independent parameter and bundle it with `size`.
As in, doing `"size,non_zero_data"` with values `[(0, False), (1MiB, True), (1MiB, False)]`.

So we can avoid the current `0 size + non_zero_data` combination that doesn't make sense, and would be burning the all contract `CODECOPY` without any gain.